Core/Src/ACDC_ADC.c

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file ACDC_ADC.h
+ * @author Liam Bramley
+ * @brief Implementation of ADC driver
+ * @version 0.1
+ * @date 2024-01-30
+ * @copyright Copyright (c) 2023-2024
+ * 
+ */
+
+#include "ACDC_ADC.h"
+#include "ACDC_SPI.h"
+#include "ACDC_GPIO.h"
+
+void ADC_Recieve(uint16_t *data){
+    delay(1);
```

Comment:
the Delay function is in the header file "ACDC_TIMER.h", it needs to be included if you want to use it